lib/active_record_view_model/update_operation.rb

```diff
@@ -331,14 +331,14 @@ def build_update_for_single_association(association_data, association_update_dat
       end
     end
 
-    def build_updates_for_collection_association(association_data, association_update_datas, update_context)
+    def build_updates_for_collection_association(association_data, association_update_ops, update_context)
       model = self.viewmodel.model
 
       # reference back to this model, so we can set the link while updating the children
       parent_data = ParentData.new(association_data.reflection.inverse_of, viewmodel)
 
-      unless association_update_datas.is_a?(Array)
-        raise ViewModel::DeserializationError.new("Invalid hash data array for multiple association: '#{association_update_datas.inspect}'")
+      unless association_update_ops.is_a?(Array)
```

Comment:
This check doesn't seem especially necessary now that we've fully preprocessed the association into functional update data